### DISCUSSION TITLE
Welcome to test-wallets Discussions!

### DISCUSSION THREAD

**Discussion by anhldbk** [zalopay-samples/test-wallets#2] [Announcements]
## 👋 Welcome!

If you have any problem regarding installing & using ZaloPay Test Wallets, feel free to ask questions here. We're glad to help

**Comment by hoangnguyen121** [comment]
anh cho em hỏi định danh tài khoản Zalopay sandbox như nào ạ, và thiết bị thật của em download cả Zalopay(đã có sẵn trên máy) và Zalopay sandbox thì có ảnh hưởng tới Zalopay đang sử dụng ko. Em cảm ơn!

**Comment by BigdataCIST** [comment]
@anhldbk đăng ký số điện thoại với zalopay sandbox của em đang bị lỗi như vậy thì xử lý thế nào ạ
![image](https://github.com/zalopay-samples/test-wallets/assets/103992475/12dc1060-08d0-4636-a7ef-564050ad9ed4)

**Comment by XuanTung95** [comment]
Anh ơi zalo pay QC app còn support nữa ko? Em tải app về rồi mà không thể nào liên kết ngân hàng để thanh toán được

**Comment by nghongphuong94** [comment]
Xin chào. Mình đã deposit nhiều lần nhưng vào app vẫn ko có tiền nên ko test được. Chức năng này có đang bị lỗi ko?

<img width="300" src="https://github.com/user-attachments/assets/7acfd058-0c08-4433-b514-d10b6580541e">
<img width="300" src="https://github.com/user-attachments/assets/b13964b9-4b03-42bb-aff2-a4c70060aea1">

**Reply by lamtranu** [reply]
bác nạp dc không ạ, giờ muốn thanh toán k biết lấy tiền ở đâu cả :(

**Reply by haki203** [reply]
gửi số mình bắn qua cho ít

**Reply by lamtranu** [reply]
> gửi số mình bắn qua cho ít

dạ 0814459969 ạ, em cảm ơn bác, mà bác kiếm dc ở đâu vậy ạ, không ấy em xin fb bác cho tiện liên lạc được khoogn ạ

**Reply by haki203** [reply]
b check thử

**Reply by lamtranu** [reply]
> b check thử

mình nhận được rồi, cảm ơn bạn nhiều ạ, mà sao b có tiền vậy :v

**Reply by haki203** [reply]
https://docs.zalopay.vn/v2/start/ ở mục A2

**Reply by lamtranu** [reply]
> https://docs.zalopay.vn/v2/start/ ở mục A2

ơ thế mà mình nạp vào k được @@
